packages/@react-spectrum/s2/stories/ActionButton.stories.tsx

```diff
@@ -20,18 +20,17 @@ import './unsafe.css';
 const meta: Meta<typeof ActionButton> = {
   component: ActionButton,
   parameters: {
-    layout: 'centered'
-    // TODO: uncomment when baseline for new S2 chromatic stories is accepted since these are resused in the chromatic stories
-    // chromatic: {
-    //   disableSnapshot: true
-    // }
+    layout: 'centered',
+    chromatic: {
+      disableSnapshot: true
```

Comment:
do we still need this `disableSnapshot` if we don't plan to run chromatic on the normal docs stories?

Reply:
I don't think we should need it anymore